packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx

```diff
@@ -0,0 +1,21 @@
+import { BodyText, Button } from "../../design-system";
+
+type Props = {
+  isVerified: boolean;
+  onButtonClick: () => void;
+};
+
+export function MyOpenPool({ isVerified, onButtonClick }: Props) {
+  return (
+    <div className="grid place-items-center bg-green-100 pt-[60px] pb-[48px]">
```

Comment:
```suggestion
    <div className="grid place-items-center bg-green-100 pt-60px pb-48px">
```

:P Ref: https://tailwindcss.com/docs/padding